### ISSUE TITLE
Review website meets accessibility standards and screenreader compatibility 

### ISSUE THREAD

**Issue by rediaz111** [fraser-lab/fraser-lab.github.io#38]
Need to run website through accessibility "linter"

**Comment by bbarad** [comment]
I expect this should be a pretty easy one to close, one of the advantages of bootstrap is that it has a lot of accessibility built in. Still worth the periodic checkup!
